distribution/lib/Standard/Tableau/0.0.0-dev/THIRD-PARTY/NOTICE

```diff
@@ -1,12 +1,2 @@
 Enso
 Copyright 2020 - 2025 New Byte Order sp. z o. o.
-
-'jna', licensed under the Apache-2.0, is distributed with the Tableau.
-The license file can be found at `licenses/APACHE2.0`.
-Copyright notices related to this dependency can be found in the directory `net.java.dev.jna.jna-5.14.0`.
```

Comment:
I think Tableau package still depends on JNA, just the distribution 'scheme' has changed.

I'd try this patch to try to make sure the license tool picks it up:
```diff
diff --git a/build.sbt b/build.sbt
--- a/build.sbt    (revision 7b7890dc5e0c76e10516a324b3f3a2679731600c)
+++ b/build.sbt    (date 1746457682075)
@@ -184,7 +184,7 @@
     "Microsoft",
     Distribution.sbtProjects(`std-microsoft`)
   ),
-  makeStdLibDistribution("Tableau", Distribution.sbtProjects(`std-tableau`))
+  makeStdLibDistribution("Tableau", Distribution.sbtProjects(`std-tableau`, `jna-wrapper`))
 )
 
 GatherLicenses.licenseConfigurations := Set("compile")
```

Reply:
Interesting idea. Not a one I would pursue. This is its realization: 30f04d9